graph/flow/mappings/common/deposit-stream.ts

```diff
@@ -24,8 +24,16 @@ export function handleDepositFlowStream(event: ethereum.Event, params: Params.De
   const availableAmount = scale(stream.availableAmount, stream.assetDecimalsValue);
 
   const now = event.block.timestamp;
-  const elapsedTime = now.minus(stream.lastAdjustmentTimestamp);
-  const snapshotAmount = stream.snapshotAmount.plus(stream.ratePerSecond.times(elapsedTime));
+
+  let snapshotAmount = stream.snapshotAmount;
+  // If the stream has not started yet, the snapshot amount is not updated.
+  if (now.gt(stream.startTime)) {
+    const streamingStart = stream.lastAdjustmentTimestamp.gt(stream.startTime)
```

Comment:
Same as for Envio - let's rename this to either `actualStartTime` or `resolvedStartTime`.